resolvers_settings_sample.yaml

```diff
@@ -58,7 +58,7 @@ polygon:
 #    gasLess: false
 #    transferAmountWei: 1000000000000000000
 #    rhsSettings:
-#      mode: { None | OffChain | OnChain }
+#      mode: { None | OffChain | OnChain | All}
```

Comment:
when is this value used if the mode is specified when identity/credentials are created?

Reply:
Based on these values we says what mode supported by the issuer. In the `IsCredentialStatusTypeSupported` method we check if param `CredentialStatusType` that was passed by user supported by issuer (for that chain).

ex: issuer supports only OffChain, than if user passes `Iden3OnchainSparseMerkleTreeProof2023` we show 400 error.